src/hssm/likelihoods/analytical.py

```diff
@@ -220,7 +199,7 @@ def logp_ddm(
     z: float,
     t: float,
     err: float = 1e-15,
-    k_terms: int = 20,
+    k_terms: int = 7,
     epsilon: float = 1e-15,
```

Comment:
I don't know what was used for testing / is used as actual value for inference, but I guess it is this default?

The epsilon for the `rt` part should rather be on the order of 1e-3, or even 1e-2.

If we are reusing the same epsilon in multiple places, we should probably separate it out.

Reply:
Was playing around. It seems that changing `k_terms` to 7 did not improve speed or computation